presto-docs/src/main/sphinx/sql/grant.rst

```diff
@@ -48,5 +48,4 @@ See connector documentation for more details.
 See Also
 --------
 
-:doc:`revoke`
-:doc:`show-grants`
+:doc:`revoke`, :doc:`show-grants`
```

Comment:
Maybe we could have bullet list here instead?

Reply:
I’ll try that out and see how it looks. We’d need to update all the pages to be consistent, so I’ll merge this fix for now.